docs/configuration.asciidoc

```diff
@@ -616,6 +616,22 @@ IMPORTANT: Use of `ServerUrls` is deprecated. Use `ServerUrl`.
 | `http://localhost:8200` | String
 |============
 
+[float]
+[[server-use-windows-credentials]]
+==== `ServerUseWindowsCredentials`
+
+[options="header"]
+|============
+| Environment variable name | IConfiguration or Web.config key
+| `ELASTIC_APM_SERVER_USE_WINDOWS_CREDENTIALS` | `ElasticApm:ServerUseWindowsCredentials`
```

Comment:
Yeah, that makes sense. Somehow, when I first wrote it, I wanted to be more descriptive, but UseWindowsCredentials sounds better to me too, because it resembles the "under-the-hood" property of the dotnet HttpClientHandler "UseDefaultCredentials" better.

Reply:
renamed all the variables/props dropping the Server prefix